test/specs/library_test.dart

```diff
@@ -45,7 +45,7 @@ void main() {
         ),
         equalsDart(r'''
           // Generated by foo!
-          //
```

Comment:
It seems like there should still be a way to get this output. Not sure how though... `comments` could be a list of nullable strings, and then null means something different from a blank string. Or a string that is only whitespace vs blank? I can't think of anything great...

Reply:
I don't know that it's an important use case to support `//` lines? But the easiest and most clear way would likely be by supporting null strings.

An alternative approach would be to support line wrapping. So, multiple lines would mean multiple groups of comments, with each line automatically wrapped at ~80 cols.

I may implement the 1st solution as that seems like the smallest API diff.

Reply:
Well, I don't have any examples, but I can imagine that it's more common to write a multi-paragraph comment (with blank `//` lines) than adjacent comments. Like if you have two paragraphs, or an example, like a code block separated from test.